src/main/java/org/myrobotlab/service/InMoovTorso.java

```diff
@@ -121,12 +122,15 @@ public boolean connect(String port) throws Exception {
       return false;
     }
 
+    /** FIXME - connections must be done externally - this
+     * needs to be a ServoController

```

Comment:
same issue as above.. test if the arduino is an instance of Arduino .. and then call connect on it.  o/w this isn't backwards compatible.

Reply:
fixed